### ISSUE TITLE
Install documentation and Python 3 support

### ISSUE THREAD

**Issue by mcgibbon** [SciTools/cf-units#117]
Currently [INSTALL](https://github.com/SciTools/cf-units/blob/master/INSTALL) says that `cf-units` is not compatible with Python 3. Looking at the code and travis builds, it looks like that's no longer the case, and that the install documentation should be updated.

Can `cf-units` be installed with conda-forge on Python 3? I tried and it looks like it's unavailable there.

**Comment by ocefpaf** [comment]
This works,

```
conda create --name TEST --channel conda-forge python=3 cf_units
```

**Comment by mcgibbon** [comment]
Ah thanks, you are correct. `conda install cf_units` works. Before I was using `conda install cf-units`, as the documentation suggests (which should also be updated).

**Comment by bjlittle** [comment]
Closed by #122 :+1:

**Comment by pelson** [comment]
Just to clarify here, to install cf-units from conda-forge it is:

```
conda install -c conda-forge cf-units
```

The ``cf_units`` name is the old form that will eventually be removed from conda-forge (depending on removal policy). 

To confirm: Our README correctly reflects this.

The rename took place around June 2018, so maybe you got caught up in the middle of the migration on conda-forge.
